### ISSUE TITLE
Adding more instance images and Resume training

### ISSUE THREAD

**Issue by pravbk100** [TheLastBen/fast-stable-diffusion#2877]
I have trained model for 3k steps with text encoder at 300. Now i want to add more instance images and resume training of this model. Should i remove the old images? And should i retrain the text encoder?

**Comment by TheLastBen** [comment]
resuming training with different datasets is tricky and usually doesn't yield the desired results, so it's up to you to see if retraining or removing old datasets helps with the desired output, it really depends on the trained subjects.

**Comment by qudabear** [comment]
Incorrect. Resuming training with new images absolutely can improve a model, but you should not re-train the text encoder. When training, you can remove problematic images and input better ones, but don't re-train the ones you already trained unless they are under-trained. Remove old images, add new ones, train again. The problem though, is the fast method often breaks the model if you re-train, so you'll need to resume with a higher learning rate than your initial learning rate, less steps.
